test/phpunit.xhprof.xml

```diff
@@ -1,39 +0,0 @@
-<phpunit backupGlobals="false"
```

Comment:
We don't need it anymore?

Reply:
bench-marking is disabled since years, and the tests also mention "remove this".
If you check the XML file: the path is even pointing to a not-existing file (`lib/Elastica/Tests` does not exist)